validations/course.validation.js

```diff
@@ -0,0 +1,18 @@
+const { body, param, header } = require("express-validator");
+const { materialType } = require("../utils/materialType");
+
+exports.CourseCreationValidation =  [
+    body("title").trim().notEmpty.withMessage("title is required"),
+    body("thumbnail").isString().withMessage("thumbnail must be a string").optional(),
+    body("objectives").isString().withMessage("objectives must be a string").optional(),
+    body("outlines").isArray().withMessage("outlines must be an array"),
```

Comment:
`isArray()` will not work here because it is an object that will be sent remove it